cli/azd/cmd/templates.go

```diff
@@ -125,11 +125,22 @@ func (tl *templateListAction) Run(ctx context.Context) (*actions.ActionResult, e
 		return nil, err
 	}
 
+	// get clickable link for a repo path
+	clickableRepoUrl := func(repoPath string) string {
+		url, err := templates.Absolute(repoPath)
```

Comment:
Instead of ignoring the error, add a `log.printf` for error and continue

Reply:
done